components/consumers/pdf/main.go

```diff
@@ -85,11 +86,20 @@ func sendToS3(filename, bucket, region string) error {
 		return fmt.Errorf("unable to upload %q to %q: %w", filename, bucket, err)
 	}
 
+	_, err = uploader.Upload(&s3manager.UploadInput{
+		Bucket: aws.String(bucket),
+		Key:    aws.String(strings.Replace(filename, ".html", "", -1) + ".pdf"),
+		Body:   bytes.NewReader(pdfBytes),
+	})
+	if err != nil {
+		return fmt.Errorf("unable to upload %q to %q: %w", filename, bucket, err)
```

Comment:
according to Go docs for fmt directives `%q	a single-quoted character literal safely escaped with Go syntax.` . filename and bucket are simple strings though